### DISCUSSION TITLE
Why do buttons have `translateY` animations by default?

### DISCUSSION THREAD

**Discussion by badalsaibo** [mantinedev/mantine#3858] [Feedback]
I think we should provide a way to disable it or override it. 

Other click animations could also be provided for easy switching. 
Like scale based animation for simulating pressed effect

**Comment by rtivital** [comment]
https://mantine.dev/theming/theme-object/#activestyles

**Reply by SyimykMitalipov** [reply]
Thanks!

**Comment by badalsaibo** [comment]
Cool. We could consider adding that info to the Button docs too!
